### DISCUSSION TITLE
Not Everyone in SharePoint is searchable

### DISCUSSION THREAD

**Discussion by WhittMatt1974** [microsoft-search/pnp-modern-search#3529] [Q&A]
We are temporarily using search as a people search.  A few individuals are not popping up in the search even though they are accessing the site. If I use the Microsoft search they are showing up but they are not showing up in the App. 

What they should see:
<img width="363" alt="modernsearch results" src="https://github.com/microsoft-search/pnp-modern-search/assets/14282089/86b8f4eb-ce42-4521-91a8-94d7c451a0bd">

What they are seeing 
<img width="561" alt="pnpSearchresult" src="https://github.com/microsoft-search/pnp-modern-search/assets/14282089/05d2d90d-89ee-415b-9eff-17b965d656a1">

### Check existing issues before logging new ones
If you are not using the latest version, triage your issue with the latest version before logging as we won't provide patches to older versions.

### Issues related to data mapping, web part setup etc, and SharePoint columns to crawled properties to managed properties should be asked in the discussion section.

**For any question regarding Web Parts usage, please use the ["Discussions"](https://github.com/microsoft-search/pnp-modern-search/discussions) tab instead of creating a new issue.**.

__Remove the above section before submitting, and removal of the below template will result in the issue being closed.__

**Version used**
Ex: 4.8

**Describe the bug**
A clear and concise description of what the bug is. Keep questions and bugs related to the web parts. If the issue relates to KQL or question around crawled and managed properties please use [Microsoft TechCommunity](https://techcommunity.microsoft.com/t5/sharepoint-developer/bd-p/SharePointDev). Which means, if you experience the same behavior in classic web parts, then it's not an issue with PnP Modern Search 😉

**To Reproduce**
Detailed steps to reproduce the behavior:
4. Enter setting A, B, C
5. See error

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Additional context**

**Comment by gretchunkim** [comment]
I suggest you use graph explorer as a testing tool to see if the problem is reproducible. Provide  a query string (user's last name for your case) and see if you are getting the same return results.  If they match (native people search and graph explorer result), then  it's likely configuration issue in the PnP modern search (match the query string  in user email, last name, first name, etc). 

(https://graph.microsoft.com/v1.0/search/query)

**Comment by kasperbolarsen** [comment]
Please follow the issue template when creating an issue.

Right now we can't assist you at all, as you haven't even provided us with info about which data source you are using!

**Comment by kasperbolarsen** [comment]
Closed due to lack of response
